doc/nrf/protocols/bt/index.rst

```diff
@@ -10,8 +10,10 @@ used world wide and supports a wide range of use cases.
 Nordic Semiconductor products support the power efficient Bluetooth LE protocol. The nRF Connect SDK  provides
 qualified Bluetooth core stack, profiles and application examples for typical use cases.
 
+If you want to go through an online training course to familiarize yourself with Bluetooth Low Energy and the development of Bluetooth LE applications, enroll in the `Bluetooth LE Fundamentals course`_ in the `Nordic Developer Academy`_.
+
 The following section describes Bluetooth solution areas and architecture.
-It also contains descriptions of Bluetooth LE Controller and Bluetooth Mesh, including the guidelines on how to qualify a product that uses these subsystems.
+It also contains descriptions of Bluetooth Mesh, including the guidelines on how to qualify a product that uses these subsystems.
```

Comment:
For Lines 15 and 16:
```
The following section describes Bluetooth solution areas and architecture, as well as the Bluetooth Mesh protocol.
It also includes guidelines on how to qualify a product that uses Bluetooth technology.
```
Since the sentence no longer has multiple subsystems, I think it makes more sense to group the information a bit differently.